proxysql/setup.py

```diff
@@ -0,0 +1,51 @@
+from codecs import open  # To use a consistent encoding
+from os import path
+
+from setuptools import setup
+
+HERE = path.dirname(path.abspath(__file__))
+
+# Get version info
+ABOUT = {}
+with open(path.join(HERE, 'datadog_checks', 'proxysql', '__about__.py')) as f:
+    exec(f.read(), ABOUT)
+
+# Get the long description from the README file
+with open(path.join(HERE, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+
+CHECKS_BASE_REQ = ['datadog-checks-base>=4.2.0', 'PyMySQL==0.9.3']
```

Comment:
No need to specfify `PyMySQL`==0.9.3 here, as it is already on the `requirements.in file`